src/components/Dashboard/TodoList.vue

```diff
@@ -883,6 +883,8 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
     // stop this cycle after 5 iterations
     if (++count > 5) {
       this.inProcessFiling = null
+      // eslint-disable-next-line no-console
+      console.log(`filing ${id} is not completed after 5 iterations`)
```

Comment:
This seems like a reasonable message to help with debugging, even in Prod.

Reply:
Helpful!

Reply:
Thanks. For someone who know what the code is doing, this is not needed. For someone who doesn't know (or forgot!) what the code is doing, this tells you the result.